apps/server/src/modules/school/domain/do/school.ts

```diff
@@ -1,43 +1,78 @@
+import { ValidationError } from '@shared/common';
 import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { LanguageType } from '@shared/domain/interface';
 import { EntityId, SchoolFeature, SchoolPurpose } from '@shared/domain/types';
-import { FileStorageType, SchoolPermissions } from '../type';
+import { FileStorageType, InstanceFeature, SchoolPermissions } from '../type';
 import { County } from './county';
 import { FederalState } from './federal-state';
 import { SchoolYear } from './school-year';
 
+interface SchoolLogo {
+	dataUrl?: string;
+	name?: string;
+}
+
 interface SchoolInfo {
 	id: EntityId;
 	name: string;
 	language?: string;
-	logo_dataUrl?: string;
-	logo_name?: string;
+	logo?: SchoolLogo;
 }
 
 export class School extends DomainObject<SchoolProps> {
+	private _instanceFeatures: Set<InstanceFeature> = new Set();
```

Comment:
Ich hab nochmal über diese Stelle nachgedacht und die gefällt mir nicht, weil die privaten Props ja eigentlich in dem Interface SchoolProps stehen. Jetzt hast du aber zwei verschiedene Stellen, wo sie stehen. Kannst du das nicht zum Interface hinzufügen?